src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comment
+};
+
+export const CommentInfo: React.FC<Props> = ({
+  comment: {
+    id, name, email, body,
+  },
+}) => (
+  <li className="CommentInfo" key={id}>
```

Comment:
The 'key' prop should not be used in this component. Instead, it should be added to the component when used in a list 🗝️